src/Core/Animation/Skeleton.cpp

```diff
@@ -223,7 +225,7 @@ Vector3 Skeleton::projectOnBone( int boneIdx, const Ra::Core::Vector3& pos ) con
     CORE_ASSERT( length_sq != 0.f, "bone has lenght 0, cannot project." );
 
     // Project on the line segment
-    const Scalar t = Math::clamp( op.dot( dir ) / length_sq, (Scalar)0.0, (Scalar)1.0 );
+    const Scalar t = std::clamp( op.dot( dir ) / length_sq, (Scalar)0.0, (Scalar)1.0 );

```

Comment:
Don't use old cast style.
```suggestion
    const Scalar t = std::clamp( op.dot( dir ) / length_sq, Scalar(0), Scalar(1) );
```

Reply:
Well your right, but it was as is (btw I accept your suggestion), but I will not review all code to check for that.

Reply:
Sure, I just wanted to avoid this new addition.

Reply:
(for the record, it was like that, so it's not an addition ...)